source/install/self-managed-changelog.md

```diff
@@ -3,13 +3,115 @@
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
-Lastest Mattermost Releases:
+Latest Mattermost Releases:
+- [Release v6.2 - Feature Release](#release-v6.2-feature-release)
 - [Release v6.1 - Feature Release](#release-v6.1-feature-release)
 - [Release v6.0 - Feature Release](#release-v6.0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
-- [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 
+## Release v6.2 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-12-16**
+
+### Important Upgrade Notes
+ -  Channel results in the channel autocomplete will include private channels. Customers using Bleve or Elasticsearch for autocomplete will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete.
```

Comment:
To clarify, this is an optional step and they can safely upgrade if they don't want private channels showing up in the autocomplete?

Is there a link to steps on how to disable the autocomplete and run the index?

Reply:
Bleve docs link: https://docs.mattermost.com/deploy/bleve-search.html
Elasticsearch docs link: https://docs.mattermost.com/scale/elasticsearch.html

Both pages include enable/disable/reindex instructions.

Reply:
> To clarify, this is an optional step and they can safely upgrade if they don't want private channels showing up in the autocomplete?

Unfortunately, I don't believe this is optional. This feature changes some internal code structures which might affect any channel autocomplete, and results might be incorrect. Although I I don't remember specifically because I didn't specifically test public channels without actually reindexing the data.

To be on the safe side, I'd recommend to reindex.

Reply:
@jasonblais only channels that the user is a member of will show up in the autocomplete. On the receiving user end, the channel will only appear as a link if the receiving user is also a member of that channel.

@amyblais lets add a clarifying statement "Note that only channel members will see private channel names in autocomplete results."

Reply:
Suggested text with edits:

"Channel results in the channel autocomplete will include private channels. Customers using [Bleve](https://docs.mattermost.com/deploy/bleve-search.html) or [Elasticsearch](https://docs.mattermost.com/scale/elasticsearch.html) for autocomplete will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete. Note that only channel members will see private channel names in autocomplete results."

Reply:
Updated text looks good. We should also include in upgrade notes since this is a recommended/required step for anyone using Bleve or Elasticsearch.